### PR TITLE
Feature/arms

### PR DESCRIPTION
**Pull request by OliviaRS**
Most of this is Skyler's work (who isn't even on GitHub??) except for the last few commits, those are mine. Take a quick pass when you can. Thanks.

### CODE REVIEW

File: Debug/src/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+*.o
```
Comment (duanem) [inline Debug/src/.gitignore:1]: We only need one git ignore file at the root directory. I'll bet smart git or eclipse is helping with these. 

You only need one.

File: src/Commands/ArmOn.cpp [hunk 2]
```diff
@@ -0,0 +1,41 @@
+#include "ArmOn.h"
+
+ArmOn::ArmOn(double speed)
+{
+mspeed = speed;
```
Comment (duanem) [inline src/Commands/ArmOn.cpp:5]: Indent

File: src/Commands/ArmOn.cpp [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+	// Use Requires() here to declare subsystem dependencies
+	// eg. Requires(chassis);
+	Requires(Robot::arms.get());
```
Comment (duanem) [inline src/Commands/ArmOn.cpp:8]: Tabs to 2 spaces

File: src/OI.cpp [hunk 4]
```diff
@@ -31,6 +32,12 @@ OI::OI() {
     dB.reset(new JoystickButton(driver.get(), 2));
     dB->WhenReleased(new ToggleShifter());
 
+    dX.reset(new JoystickButton(driver.get(), 3));
```
Comment (duanem) [inline src/OI.cpp]: We could use some symbolic names for these 2,3&4.

File: src/OI.cpp [hunk 5]
```diff
@@ -31,6 +32,12 @@ OI::OI() {
...
+    dX->WhenReleased(new ArmOn(1.0));
```
Comment (duanem) [inline src/OI.cpp]: WhenReleased doesn't seem right. WhenPressed?

**Review by duanem — COMMENTED** [review]
FTP

Finished this pass.

There's a few comments that should be fixed.

**Comment by duanem** [comment]
This is probably good enough to commit. Tuning the PID can come after this.

LGTM

Please merge.

**Comment by duane701** [comment]
Closing in favor of #7.

**Pull request closed without merging**